org-formation/organization.yaml

```diff
@@ -72,23 +75,35 @@ Organization:
       Accounts:
         - !Ref ITSandboxAccount
 
-  SynapseOU:
+  SynapseProdOU:
     Type: OC::ORG::OrganizationalUnit
     Properties:
-      OrganizationalUnitName: synapse
+      OrganizationalUnitName: SynapseProd
       Accounts:
-        - !Ref SynapseDevAccount
         - !Ref SynapseDWAccount
         - !Ref SynapseProdAccount
 
-  BridgeOU:
+  SynapseDevOU:
```

Comment:
These new OUs only inherit SCPs from the ORG root.  Did you want to apply any additional SCPs to these?

Reply:
I don't know of different SCPs we would want to deploy today, but this would let us do that in the future.